locales/pl.json

```diff
@@ -872,7 +903,7 @@
       "fail": "W HTML 5, atrybuty scope mogą być używane tylko w elementach nagłówkowych tabeli (<th>)."
     },
     "same-caption-summary": {
-      "pass": "Treść atrybutu summary i <caption> nie są powielane.",
+      "pass": "Treści atrybutu summary i <caption> nie są powielane.",
```

Comment:
```suggestion
      "pass": "Treści atrybutu summary i elementu <caption> nie są powielane.",
```

Reply:
@ajanec01 : Thank you for your valuable comments. I will add the revised version again tomorrow. I have made a few other minor changes to it. Can I ask you to review it again tomorrow?

Reply:
Sure!